lib/bulk/common.js

```diff
@@ -208,7 +211,7 @@ var BulkWriteResult = function(bulkResult) {
  * @property {string} errmsg Write concern error message.
  * @return {WriteConcernError} a WriteConcernError instance
  */
-var WriteConcernError = function(err) {
+const WriteConcernError = function(err) {
```

Comment:
let's make all of these just functions, not expressions. I won't comment on the rest